docs/source/technical_tutorials/access/copc_access.ipynb

```diff
@@ -0,0 +1,291 @@
+{
```

Comment:
<p>This output doesn't appear to match the code. <code>maap.downloadGranule</code> does not produce such output, but <code>earthaccess.download</code> does. Am I missing something? Did maap-py change to use earthaccess under the covers?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/459/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>